src/pages/node-express/lesson10-m1.mdx

```diff
@@ -0,0 +1,46 @@
+---
+layout: "../../layouts/genericMarkdownFile.astro"
+title: "Node/Express Lesson 10: Jobs API Part 2"
+description: Material link for lesson 10 of the node/express class
+---
+
+import {
+  AssignmentLinkButton,
+  NextLinkButton,
+  PrevLinkButton,
+} from "../../components/navButtons.js";
+
+# Node/Express Lesson 10
+
+# Jobs API Part 2
+
+In this lesson, you complete implementation of the CRUD operations for the API, using either the `Job` model as the instructor does, or the model you invented in the last lesson. You test with Postman as you go. Your Postman configuration should have environment variables for the URL and, as the instructor explains, the accessToken. You will improve the error handling to return meaningful error messages to the user when Mongoose validation errors occur. You then add security protection for the application so that it can be deployed on the Internet. The security configuration uses the following node packages:
+
+- [helmet](https://www.npmjs.com/package/helmet) for setting various HTTP headers to protect against well-known vulnerabilities
+- [cors](https://www.npmjs.com/package/cors) for enabling cross-origin resource sharing
+- [xss-clean](https://www.npmjs.com/package/xss-clean) for preventing cross-site scripting attacks. Note that this package is now deprecated. We can still use it for this assignment, but we will need to find a replacement for it in the future.
+- [express-rate-limit](https://www.npmjs.com/package/express-rate-limit) for limiting the number of requests from a single IP address. This is to prevent denial of service attacks.
```

Comment:
Added links and short blurb (there's existing content below that goes more into depth about these).